README.md

```diff
@@ -10,6 +10,15 @@ This is a little helper script that helped us with organizing the GNU/Linux inst
 - Extracts emails from the list.
 - Extracts the emails of those who opted in for the feedback form.
 
-### TODO: 
+### Usage guidelines:
+- Clone (`git clone https://github.com/cs-unipi-FOSS/InstallfestHelper.git`) or download as zip.
+- Move to InstallfestHelper folder (`cd /path/to/InstallfestHelper`).
+- Allow installfest.py as executable (`chmod +x installfest.py`).
+- Run `python installfest.py CSVfile` where _CSVfile_ is your input file.
+- Run `python installfest.py -h` or `python installfest.py --help` to see the help message.
+- The parameter _--get-feedback-recipients_ returns a list with all the feedback from recipients.
```

Comment:
The `--get-feedback-recipients` parameter returns a list with the emails of all those who opted in for feedback, not the feedback itself.